scripts/lis_hyperv_platform/setup_env.ps1

```diff
@@ -1,21 +1,34 @@
 param (
-    [String] $JobPath = 'C:\var\lava\tmp\1',
+    [String] $JobPath = "C:\path\to\job",
     [String] $VHDPath = "C:\path\to\example.vhdx",
     [String] $UserdataPath = "C:\path\to\userdata.sh",
     [String[]] $KernelURL = @(
         "http://URL/TO/linux-headers.deb",
         "http://URL/TO/linux-image.deb",
         "http://URL/TO/hyperv-daemons.deb"),
     [String] $InstanceName = "Instance1",
-    [String] $MkIsoFS = "C:\path\to\mkisofs.exe"
+    [String] $MkIsoFS = "C:\path\to\mkisofs.exe",
+    [String] $LavaToolsDisk = "C:\path\to\tools"
 )
 
+function MyTest-Path {
```

Comment:
Get rid of this method and make your code check for output values.
Writing code with side effects makes it very hard to read.

Reply:
If you do want to keep it, make sure it is defined in one place and one place only.
And the naming should be changed to something like:
Assert-PathExists, as you are doing a kind of assertion.